democracy_club/apps/tests/test_urls.py

```diff
@@ -0,0 +1,58 @@
+import pytest
```

Comment:
The content of this looks good but I would rethink where this file is stored and the naming.

Adding a new `tests` app for a single file is a bit overkill and possibly a bit misleading - as you might then expect all tests to be in here. It would be better to keep the file in the `core` app, especially as the tests span multiple apps.

Also the name of the file `test_urls.py` is misleading. We do build a list of urls that then we use in the test, but it is not the urls themselves we are testing, it is the HTML content returned. So `test_html.py` is more representative.

Reply:
Thanks for the reminder. Moved & renamed.